ui/packages/tidb-dashboard-for-clinic/src/client/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
```

Comment:
Like swagger specs, we still need many duplicated codes between project `x` and dashboard-for-x.